### DISCUSSION TITLE
Using AWS CodeCommit as helm repo for Argo CD

### DISCUSSION THREAD

**Discussion by Sir-gits-a-lot** [argoproj/argo-cd#15537] [General]
I am trying to deploy to Argo CD using helm chart tar.gz stored in Code Commit Repository.
But it doesn't seem to work. Although when I store Plain K8s manifests in the same Repository, it deploys to the local cluster.

If anybody has any idea, please let me know.

**Comment by jannfis** [comment]
Hi, you are not supposed to store a packaged Helm chart in Git, but rather an unpacked version of it.

Please have a look at our example application here: https://github.com/argoproj/argocd-example-apps/tree/master/helm-guestbook
